Sources/HipMobileUI.Droid/Contracts/AndroidFabSizeCalculator.cs

```diff
@@ -30,7 +31,7 @@ public int CalculateFabSize()
                 return 56;
             }
             // compat libraries use pixel, therefore convert dp to pixel
-            return (int) (56 * scale + 0.5);
+            return (int) (56 * density * 0.5);
```

Comment:
Why times `0.5`? Converting from `dp` to `px` should just be according to `px = dpValue * density`, if I remember correctly.

Reply:
You were right. This calculation was just randomly correct for dpi value of 480, but the px value returned by 56 * density was wrong for all kind of dpi values. I have completly changed the way how the button is positoned for android versions below Lollipop and tested my new implementation for nearly all dpi values.